xds/internal/xdsclient/transport/transport.go

```diff
@@ -465,7 +475,7 @@ func (t *Transport) recv(stream adsStream) bool {
 			stream:  stream,
 			version: rVersion,
 		})
-		t.logger.Infof("Sending ACK for resource type: %v, version: %v, nonce: %v", url, rVersion, nonce)
+		t.logger.Debugf("Sending ACK for resource type: %q, version: %q, nonce: %q", url, rVersion, nonce)
```

Comment:
Is this debug? Im trying to think, since we are logging a WARN for NACKs, should we use the same philosophy for ACKs?

Reply:
`WARNING` for NACKs since they indicate a problem which might not be a fatal one. As far as ACKs are concerned, if things are working fine, as the application owner, you really dont way to see any logs.

Reply:
In fact, the amount of logs that we have been throwing when things are working well, is so huge that some of our storage teams are getting throttled by cloud logging, and then when things go bad, they end up losing those logs.